myRBM/test_label_input.m

```diff
@@ -39,7 +39,8 @@
         energies(end + 1) = sum(energy);
     end
     
-    valPred = sum(vk(785:812))/length(vk(785:812));
+    valPred = round(sum(vk(785:812))/length(vk(785:812)),4)*10; % predicted valence
```

Comment:
having free numbers in the middle of code isn't great because it reduces the readability (known as magic numbers: https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad ). What do these numbers mean? if possible I would re-write this so it's clear where '785', '812' , '4', etc. are coming from.